{
	"id": "CVE-2022-35935",
	"tags": [
		{
			"label": "Product",
			"message": "tensorflow"
		},
		{
			"label": "Version",
			"message": "n/a"
		},
		{
			"label": "Vulnerability",
			"message": "CWE-617: Reachable Assertion"
		}
	],
	"description": " TensorFlow is an open source platform for machine learning. The implementation of SobolSampleOp is vulnerable to a denial of service via CHECK-failure (assertion failure) caused by assuming `input(0)`, `input(1)`, and `input(2)` to be scalar. This issue has been patched in GitHub commit c65c67f88ad770662e8f191269a907bf2b94b1bf. The fix will be included in TensorFlow 2.10.0. We will also cherrypick this commit on TensorFlow 2.9.1, TensorFlow 2.8.1, and TensorFlow 2.7.2, as these are also affected and still in supported range. There are no known workarounds for this issue.",
	"poc": {
		"githubs": [
			"https://github.com/ARPSyndicate/cvemon",
			"https://github.com/skipfuzz/skipfuzz"
		]
	}
}
